accountability_project/Models/spaces/api/serializers.py

```diff
@@ -68,6 +68,7 @@ class SpaceSerializerToReadWithHabits(serializers.ModelSerializer):
     members_count = serializers.IntegerField(read_only=True)
     habits_count = serializers.IntegerField(read_only=True)
     # shows the detailed habits that belong to this space 
+    members = serializers.SerializerMethodField(method_name='get_members')
```

Comment:
- [ ] Please change the name of this class to SpaceSerializerToReadWithHabitsAndMembers -> to be more explicit
- [ ] also, something like this does not work in this case? `members = SimpleUserSerializer(many=True, read_only=True)`